mlir/examples/standalone/really_alone/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+add_compile_definitions("MLIR_PYTHON_PACKAGE_PREFIX=mlir.")
```

Comment:
we need to set `-DMLIR_PYTHON_PACKAGE_PREFIX=mlir` so that `StandReallyAloneExtensionNanobind.cpp` compiles with the right [`#define`](https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir-c/Bindings/Python/Interop.h#L50-L85)s (otherwise APIs with `MlirContext` and etc. won't work).